From d45f87b96aec543d0cd685e506c7b824cc523c5b Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Wed, 28 Sep 2022 19:11:30 +0300
Subject: [PATCH] [SW-99831] gaudi3: d2d: update hw errata wa values

Modifying the hw errata workaround values for GAUDI3_0314,
the architecture team has updated their recommended values to
be as per https://jira.habana-labs.com/browse/H9-5003.

Updating the same in rom.

Change-Id: Ibeb2e4396dbee53c57fe2f59a993dac8cc6510bb
---

diff --git a/arch/gaudi3/arc/recovery.c b/arch/gaudi3/arc/recovery.c
index 235d6d3..ea99e56 100644
--- a/arch/gaudi3/arc/recovery.c
+++ b/arch/gaudi3/arc/recovery.c
@@ -145,8 +145,8 @@
 	}
 }
 
-#define PLL_LOCK_TIMER_VAL		0xC350
-#define OFFSET_CALLOCK_TIMER		0xC350
+#define PLL_LOCK_TIMER_VAL		0x1D4C
+#define OFFSET_CALLOCK_TIMER		0x61A8
 #define TIMER_1US			0x32
 #define NUM_OF_PHY			6
 #define INTER_PHY_DIST			(mmD0_DPHY1_CTRL_BASE - mmD0_DPHY0_CTRL_BASE)
